Source/Core/Core/IOS/USB/Host.cpp

```diff
@@ -175,6 +177,23 @@ void USBHost::DispatchHooks(const DeviceChangeHooks& hooks)
     OnDeviceChangeEnd();
 }
 
+void USBHost::AddEmulatedDevices(std::set<u64>& new_devices, DeviceChangeHooks& hooks)
+{
+  if (Config::Get(Config::MAIN_EMULATE_SKYLANDER_PORTAL))
+  {
+    auto skylanderportal = std::make_unique<USB::SkylanderUSB>(m_ios, "Skylander Portal");
+    if (ShouldAddDevice(*skylanderportal))
+    {
+      const u64 skyid = skylanderportal->GetId();
+      new_devices.insert(skyid);
+      if (AddDevice(std::move(skylanderportal)))
```

Comment:
```suggestion
      if (AddDevice(std::move(skylanderportal)) || always_add_hooks)
```
(at some point, we should probably refactor this function to deduplicate the `if (AddDevice(...) || always_add_hooks) add_hook(...)` logic)